src/Symfony/Component/Notifier/Bridge/Mercure/MercureOptions.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Mercure;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Mathias Arlaud <mathias.arlaud@gmail.com>
+ *
+ * @experimental in 5.3
+ */
+final class MercureOptions implements MessageOptionsInterface
+{
+    private $topic;
+
+    public function __construct(string $topic)
```

Comment:
Shouldn't we allow the user to set other Mercure options, and especially to mark updates a private?

Reply:
Yes, indeed. And maybe `$topic` should be an array of topics as well?

Reply:
Added every option!
Moreover, I've added the possibility to deal with multiple topics in options but also in the dsn